src/mpid/ch4/src/ch4r_symheap.h

```diff
@@ -48,6 +48,16 @@ static inline size_t MPIDI_CH4R_get_mapsize(size_t size, size_t * psz)
 }
 
 #undef FUNCNAME
+#define FUNCNAME MPIDI_CH4R_check_mapaddr
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static inline int MPIDI_CH4R_check_mapaddr(void *start)
```

Comment:
I followed MPIDI_CH4R_check_maprange_ok to make the symheap functions consistent. But I agree this is not easy-to-read. It is not intuitive to me as well... I will make the change.